vendor/bat-native-ledger/src/bat_client.cc

```diff
@@ -374,12 +374,12 @@ void BatClient::recoverWalletCallback(bool result,
   }
   ledger_->SetDays(days);
   ledger_->SetWalletProperties(properties);
-  ledger_->SetPaymentId(recoveryId);
+  ledger_->SetPaymentId(recovery_id);
   ledger_->OnRecoverWallet(ledger::Result::LEDGER_OK, properties.balance_, properties.grants_);
 }
 
-void BatClient::getGrant(const std::string& lang, const std::string& forPaymentId) {
-  std::string paymentId = forPaymentId;
+void BatClient::getGrant(const std::string& lang, const std::string& for_payment_id, const std::string& safetynet_token) {
```

Comment:
also lint issues here for line length. Should be <= 80 chars

Reply:
Aligned code according to lint requirements